docs/README.md

```diff
@@ -44,6 +45,10 @@ The following JSON is the default configuration for the module and can be overri
 }
 ```
 
+## Developmnt Mode
+
+Yoast SEO includes a _development mode_, this is enabled by default for non-production environments and will _pretty print_ the _yoast-schema-graph_ HTML rather than the minified default.
```

Comment:
Not sure the italics are really necessary here, and I'd probably just refer to the JSON+LD schema.org graph output. It's not a Yoast invention 😛

Reply:
```suggestion
Yoast SEO includes a development mode, this is enabled by default for non-production environments and will pretty print the JSON+LD schema output rather than the minified default.
```